docs/_docs/Develop/script-js.md

```diff
@@ -200,6 +200,13 @@ var Installer = Java.extend(org.phoenicis.scripts.Installer, installerImplementa
 ### Advanced
 This section describes some advanced methods which give you more possibilities to control your script.
 
+### Include mechanism
+When you want to use a certain functionality in your scripts, you need to include it in your scripts, for example:
+```javascript
+include(["engines", "wine", "quick_script", "steam_script"]);
+```
+allows you to execute a steam script. The content of the include can be determined from the path of the functionality (in the scripts repository) you want to use and the id of the functionality (check the `script.json` file near the `script.js` file containing the functionality).
```

Comment:
It is not really the path leading to the functionality. More correct in my opinion is the path, made up of the ids on the different folder levels, leading to the functionality.